demisto_sdk/commands/common/hook_validations/graph_validator.py

```diff
@@ -206,14 +206,30 @@ def handle_invalid_toversion(
 
     @error_codes("GR103")
     def is_file_using_unknown_content(self):
+        """Validates that there is no usage of unknown content items.
+        The validation runs twice:
+        1. Cases where a warning should be raised - if the using content item is a test playbook/test script,
+            or if the dependency is optional.
+        2. Cases where an error should be raised - the complementary case.
+        """
+        return all(
+            [
+                self._find_unknown_content_uses(raises_error=False),
+                self._find_unknown_content_uses(raises_error=True),
+            ]
+        )
+
```

Comment:
```suggestion
        is_valid = [
                self._find_unknown_content_uses(raises_error=False),
                self._find_unknown_content_uses(raises_error=True),
            ]
        return all(is_valid)

```

Reply:
For debugging reasons, I find that it is better to do it this way (easier to see which one of the validation returned the faulty error)